graph2tac/loader/predict_server.py

```diff
@@ -230,15 +224,12 @@ def summary_log_message(self) -> str:
             "Initialize|Network build time (s)" : f"{self.build_network_time:.6f}",
             "Initialize|Def clusters to update" : f"{self.n_def_clusters_updated}",
             "Initialize|Def update time (s)" : f"{self.update_def_time:.6f}",
-            "Predict|Messages cnt" : f"{self.msg_idx+1}",
-            "Predict|Avg predict time (s/msg)" : 
-                f"{self.t_predict/self.n_predict:.6f}" if self.n_predict else "NA",
+            "Predict|Messages cnt" : f"{n_steps}",
+            "Predict|Avg predict time (s/msg)" : f"{self.t_predict/n_steps:.6f}",
             "Predict|Max predict time (s/msg)" : f"{self.max_t_predict:.6f}",
```

Comment:
Just so I know, will this be 1 if it is the first message now?

Reply:
I don't think something would be `1`, what in particular do you mean by "this"?

Reply:
Oh, sorry, I meant the line below:
```
"Predict|Max time predict msg ix" : f"{self.argmax_t_predict}",
```
If the first predict takes the longest, will it show up as 0 or 1 now?

Reply:
It being `self.argmax_t_predict`